cmd/modern/root/config/add-context.go

```diff
@@ -93,4 +94,23 @@ func (c *AddContext) run() {
 		{localizer.Sprintf("To start interactive query session"), "sqlcmd query"},
 		{localizer.Sprintf("To run a query"), "sqlcmd query \"SELECT @@version\""},
 	}, localizer.Sprintf("Current Context '%v'", context.Name))
+	c.LogTelemtry()
```

Comment:
>c.LogTelemtry() [](http://example.com/codeflow?start=1&length=15)

instead of requiring `run` implementation to call this directly, is it possible to define an interface with `LogTelemetry` as a member and have whatever calls `run` also call it if the object implements it?